unified_planning/test/test_credits.py

```diff
@@ -56,6 +56,9 @@ def test_long_env_credits(self):
         credits_printed = credits.getvalue()
         # test that every keyword occur the same number of time in the printed result
         number_of_credits_printed = credits_printed.count(credits_keywords[0])
-        for keyword in credits_keywords:
-            self.assertIn(keyword, credits_printed)
-            self.assertEqual(credits_printed.count(keyword), number_of_credits_printed)
+        if number_of_credits_printed > 0:  # If at least one credit was printed
```

Comment:
Why is this fix here?

Reply:
Because when testing with a clean python environment (on pipenv), this test was failing.